docs/architecture/cloud-native/cloud-native-data.md

```diff
@@ -1,9 +1,10 @@
 ---
-title: Database-per-microservice
+title: Cloud-native data
 description: Contrast data storage in monolithic and cloud-native applications.
 author: robvet
-ms.date: 01/22/2020
+ms.date: 04/21/2020
 ---
+
 # Database-per-microservice
```

Comment:
Why two completely different names for title and H1? Can't you keep the old file name then?

Reply:
In data chapter? thx

Reply:
Fixed